src/main/java/io/scif/formats/JavaFormat.java

```diff
@@ -92,7 +86,7 @@ public void setDest(final RandomAccessOutputStream stream,
 			IOException
 		{
 			super.setDest(stream, imageIndex, config);
-			if (getStream().length() == 0) writeHeader();
+			if (getStream().length() == 0) { writeHeader(); }
```

Comment:
Well, I guess I have to manually hack IntelliJ to these settings. The Eclipse Formatter plugin with the ImageJ settings doesn't seem to do this.